tsflex/features/segmenter/strided_rolling.py

```diff
@@ -494,8 +494,8 @@ def apply_func(self, func: FuncWrapper) -> pd.DataFrame:
         log_window = "manual" if self.window is None else self.window
         logger.info(
```

Comment:
just an idea: is there a flush option (alike with print statements)? That the logging message must be sent before the next code will get hit?

Reply:
No idea.. not really sure where you aiming at with this? :eyes:

Reply:
That the timing diff is created based on when these logging messages, are outputted; so when they are flushed, we are certain that they are outputted right-away.

Reply:
Hmm, we use the `elapsed` variable for the timing diff (which is calculated right after the function output is stored in the output dict) - I do not see any problems with this

Reply:
Flush might however facilitate more direct output towards the end user?

Reply:
No biggie! :handshake: